test/file-store.js

```diff
@@ -0,0 +1,59 @@
+/* global describe, it */
+
+const path = require('path')
+const assert = require('assert')
+
+const FileStore = require('../lib/file-store.js')
```

Comment:
For consistency, drop the `.js`.  The module does not import in this way other places.